Document cross-org query UUIDs (#36529)
DataDog/documentation: content/en/account_management/org_settings/cross_org_visibility.md

This document explains:
- What cross-organization visibility [enables](#capabilities)
- How to [share](#create-a-connection) data across your organizations

> - How to create a [Dashboard and Notebook widget](#create-a-widget-with-cross-organization-data) or API query with data from your other organizations

Reviewer: ```suggestion
- How to create a [Dashboard and Notebook widget](#create-a-widget-with-cross-organization-data) or [API query](#in-the-api-1) with data from your other organizations
```

The <a href="https://registry.terraform.io/providers/DataDog/datadog/latest/docs">Datadog Terraform Provider</a> does not support creation of cross-organization connections. However, you can manage a dashboard containing widgets with cross-org queries through Terraform by exporting the dashboard to JSON.
</div>

> Cross-organization data can be selected in Dashboard and Notebook widget JSON. The same per-query field is also available in the Metrics API formula query endpoints:

Reviewer: ```suggestion
You can include cross-organization data in Dashboard and Notebook widget JSON with the `cross_org_uuids` field. The same field is available in the Metrics API formula query endpoints:
```

Reviewer: The phrase "the same per-query field" assumes the reader has already encountered the field name (cross_org_uuids), but in the new ordering, the explicit name first appears in the next paragraph (line 112)

- [Query timeseries data across multiple products][8]
- [Query scalar data across multiple products][15]

> When you define a widget in the Dashboard API, or send a Metrics API formula query request, use the `cross_org_uuids` parameter in each query object to identify the source organization in a cross-organization query.

Reviewer: ```suggestion
When you define a widget in the Dashboard API or send a Metrics API formula query request, use the `cross_org_uuids` parameter in each query object. The parameter identifies the source organization in a cross-organization query.
```

Reviewer: This sentence is being flagged by Vale as too long. Suggesting to break it up into two sentences.
